tests/e2e/creation_test.go

```diff
@@ -444,6 +450,46 @@ func (tc *testContext) testDefaultCertsAvailable() error {
 	return nil
 }
 
+func (tc *testContext) testTrustedCABundle() error {
+	CAConfigMapName := "odh-trusted-ca-bundle"
+	CADataFieldName := "odh-ca-bundle.crt"
+
+	if tc.testDSCI.Spec.TrustedCABundle.ManagementState == operatorv1.Managed {
+		foundConfigMap := &corev1.ConfigMap{}
+		err := tc.customClient.Get(tc.ctx, client.ObjectKey{
+			Name:      CAConfigMapName,
+			Namespace: tc.testDSCI.Spec.ApplicationsNamespace,
+		}, foundConfigMap)
+
+		if err != nil {
+			return fmt.Errorf("Config map not found, %w", err)
+		}
+
+		checkNewline := strings.Contains(foundConfigMap.Data[CADataFieldName], "/n")
+
+		if checkNewline == false {
+			fmt.Print("Newline not found at the end of configmap")
+		}
```

Comment:
Are both cases acceptable here (newline or not), or should this assert one of the cases?

Also, the check looks like it will pass if a newline character is found _anywhere_ in the string, not just at the end.

Reply:
@grdryn yea true, i have modified the check here